src/launchWizard/launchWizard.js

```diff
@@ -60,9 +60,6 @@ function getConfigValues() {
   const tdmlDescription = document.getElementById('tdmlDescription').value
   const tdmlPath = document.getElementById('tdmlPath').value
   const openDataEditor = document.getElementById('openDataEditor').checked
-  const openInfosetDiffView = document.getElementById(
```

Comment:
Don't delete the infoset diff view stuff. This stuff will still be needed just updated to fix the label/text

Reply:
> Don't delete the infoset diff view stuff. This stuff will still be needed just updated to fix the label/text

Agree, We should not delete that functionality as it works as intended. @lrbarber will bring that functionality back and only fix the description mismatch along with fixing the infoset.ts for Uri encoding.